src/App.tsx

```diff
@@ -122,10 +120,14 @@ const App: React.FC = () => {
             <Route path='/how-to-buy'>
               <HowToBuy title={'Index - How to Buy'} />
             </Route>
-            <Route exact path='/join'
+            <Route
+              exact
+              path='/join'
               render={() => (window.location.href = discordLink)}
             />
-            <Route exact path='/discord'
+            <Route
```

Comment:
FYI I don't think Prettier actually runs on this codebase b/c this diff was applied when I used VSCode `CMD + P` > Format Document With... > Prettier

I've seen a handful of instances where double quotes are used when single quotes are specified:

https://github.com/SetProtocol/index-ui/blob/master/.prettierrc.json#L3